src/styles/home.styl

```diff
@@ -156,7 +156,8 @@ a.signin-link {
   top: 50%;
   left: 50%;
   width: 50%;
-  width: 90vh;
+  --diamond-size: 90vh;
+  width: var(--diamond-size);
```

Comment:
Could we get a fallback width for browsers that don't support CSS variables? Currently not supported by Edge.